projects/sky-autonumeric-showcase/src/app/visual/visuals.component.html

```diff
@@ -0,0 +1,9 @@
+<h1>
+  Visual testing
+</h1>
+
+<ul>
+  <li>
+    <a routerLink="visual/autonumeric">Autonumeric</a>
```

Comment:
`RouterModule` has been imported in the `VisualModule`, but this `routerLink` is still not working. Any ideas?

Reply:
Resolved. Was missing a declaration for the root visual component in `ngModule`.